packages/components/src/FormField/FormFieldTypes.ts

```diff
@@ -115,6 +116,26 @@ export interface FormFieldProps {
    */
   readonly value?: string | number;
 
+  /**
+   * Set the component prefix to the given prefix text.
+   */
+  readonly prefixText?: string | number;
+
+  /**
+   * Set the component prefix icon to the given icon name.
+   */
+  readonly prefixIcon?: IconNames;
+
+  /**
+   * Set the component prefix to the given prefix text.
+   */
+  readonly postfixText?: string | number;
+
+  /**
+   * Set the component to the given value.
+   */
+  readonly postfixIcon?: IconNames;
+
```

Comment:
Ideally these are only 2 props.

We want props to be as predictable and simple to the consumer as possible. If I am a consumer of the component, i would not expect to have to add `prefixIcon`, `prefixText`, `prefixAction` to have a full prefix on my component.
 
```ts
type Affix = {
  icon?: IconName
  label?: string
  action?: () => void
}

prefix?: Affix
postfix?: Affix
```